docs/src/getting_started.md

```diff
@@ -0,0 +1,189 @@
+# Getting Started with JumpProcesses in Julia
+
+Jumps (or point) processes are stochastic processes with discrete state changes
+driven by a `rate` function. Historically, jump processes have been developed in
+the context of dynamical systems to describe dynamics with sudden changes —
+the jumps — in a system's value at random times. In contrast, the development
+of point processes has been more focused on describing the occurrence of random
+events — the points — over a support. In reality, jump and point processes
+share many things in common which make JumpProcesses ideal for both.
+
+Processes involving multiple jumps are known as compound jump (or point)
+processes.
+
+```math
+du = \sum_{j} h_j(u,p,t) dN_j(t)
+```
+
+where ``N_j`` is a jump process with rate ``\lambda_i(u,p,t)``.
```

Comment:
This is too vague. You need to fully explain the equation and the meaning of all the terms in it if you are putting this forward as the new opening introduction to the package. What is the "jump process" $N_j$, what is $h_j$? What is a "rate"?

Reply:
Explanation added.